svg-aam/name/comp_host_language_label.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<html>
+<head>
+  <title>Name Comp: Host Language Label</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+<h1>SVG-AAM: Label Tests</h1>
+
+<p>Tests SVG-specific host language label rules (title element and xlink:link attr) in <a href="https://w3c.github.io/svg-aam/#mapping_additional_nd">SVG-AAM §8.1 Name and Description</a>, but note the open issues in <a href="https://github.com/w3c/svg-aam/issues/31">SVG-AAM #31</a>.
+
+
+<h2>SVG * > title</h2>
+<svg viewbox="0 0 300 100">
+  <circle cx="26" cy="26" r="25" data-expectedlabel="circle label" data-testname="circle > title" class="ex"><title>circle label</title></circle>
+  <rect x="60" y="1" width="50" height="50" data-expectedlabel="rect label" data-testname="rect > title" class="ex"><title>rect label</title></rect>
+  <polygon points="100,100 150,25 150,75 200,0" fill="none" stroke="black" data-expectedlabel="polygon label" data-testname="polygon > title" class="ex"><title>polygon label</title></polygon>
+</svg><br>
+<svg viewbox="0 0 200 90">
+  <g fill="white" stroke="green" stroke-width="5" data-expectedlabel="group label" data-testname="g > title" class="ex">
```

Comment:
Good test for `<g>`!

I see in svg-aam ([5.1.2 Including Elements in the Accessibility Tree](https://w3c.github.io/svg-aam/#include_elements)) that `aria-label`/`aria-labelledby` are also valid labelling techniques if an element meets the a11y tree inclusion criteria. Assuming we're not talking about overridden `svg`s (e.g., `<svg role="img"`), do we need to include `aria-label`/`aria-labelledby` examples? (or perhaps this note on ARIA labelling only applies to `svg` elements with other explicit role assignments?)

Reply:
Yes, this file just includes the host language label mechanisms; I followed the same algorithm file naming convention in the `/accname/name/` dir. We can definitely add more tests for aria-label with an another file: a new `/svg-aam/name/comp_label.html` could be more exhaustive like the `comp_label.html` you added in `/accname/name/comp_label.html`